### ISSUE TITLE
a11y: Fix missing label attribute issue with form input control

### ISSUE THREAD

**Issue by nwmac** [rancher/dashboard#12818]
a11y issue: 86

**Issue 86 - Missing form label or 'title' attribute**

The form field do not have the label associated with them and "title" attribute is not included in their source code either resulting in screen reader users fail to understand the purpose of the form field.

This should be fixed in the form input control(s).

This appears in several places:

Account and API Keys
 - API key section
  - Filter input field 

Projects/Namespaces
- Filter input field

Import YAML
- Type Import YAML data textarea

Example:

![image](https://github.com/user-attachments/assets/29e357ca-19d2-44d3-be9b-e015c503edd9)

Suggested remediation:

Apply the following changes:
• Provide unique and descriptive visual label for the mentioned form fields using <label> tag.
• Use "for" and "id" attributes to associate labels with their respective form fields. Ensure that the value of "for" attribute of the label exactly matches the value of "id" attribute of the form field.

**Comment by yonasberhe23** [comment]
Requires manual validation

**Comment by izaac** [comment]
Rancher version: `v2.11-13170b249a26f304b1ae89a96688f547a96b77b1-head`
UI version: `master 54c4223`

This is validated as properly working on the Filter fields, and YAML Editor getting picked by screen readers.
